classes/classes/Saves.as

```diff
@@ -876,31 +876,23 @@ public function saveGameObject(slot:String, isFile:Boolean):void
 		saveFile.data.beardLength = player.beardLength;
 		saveFile.data.eyeType = player.eyeType;
 		saveFile.data.beardStyle = player.beardStyle;
-		saveFile.data.skinType = player.skinType;
-		saveFile.data.skinTone = player.skinTone;
-		saveFile.data.skinDesc = player.skinDesc;
-		saveFile.data.skinAdj = player.skinAdj;
-		saveFile.data.faceType = player.faceType;
 		saveFile.data.tongueType = player.tongueType;
 		saveFile.data.earType = player.earType;
 		saveFile.data.earValue = player.earValue;
 		saveFile.data.antennae = player.antennae;
 		saveFile.data.horns = player.horns;
 		saveFile.data.hornType = player.hornType;
-		// <mod name="BodyParts.Skin and UnderBody" author="Stadler76">
-		saveFile.data.underBody = player.underBody;
-		// </mod>
-		// <mod name="Predator arms" author="Stadler76">
-		saveFile.data.clawTone = player.clawTone;
-		saveFile.data.clawType = player.clawType;
-		// </mod>
+		/*for each (var key:String in ["underBody","lowerBodyPart","skin","clawsPart"]) {
+			saveFile.data[key] = (player[key] as SimpleJsonable).saveToObject();
+		}*/
+		saveFile.data.face = player.facePart.saveToObject();
+		saveFile.data.underBody = player.underBody.saveToObject();
+		saveFile.data.lowerBodyPart = player.lowerBodyPart.saveToObject();
+		saveFile.data.skin = player.skin.saveToObject();
```

Comment:
Same as for the claws storing:
Why convert the saves here? You're breaking BC for the sake of what?